source/extensions/filters/http/cache/cache_policy.h

```diff
@@ -0,0 +1,68 @@
+#pragma once
+
+#include "envoy/http/header_map.h"
+#include "envoy/stream_info/filter_state.h"
+
+#include "source/extensions/filters/http/cache/cache_headers_utils.h"
+#include "source/extensions/filters/http/cache/http_cache.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace Cache {
+
+struct CacheEntryUsability {
+  CacheEntryStatus status = CacheEntryStatus::Unusable;
+  Envoy::Seconds age = Envoy::Seconds::max();
```

Comment:
We are in namespace `Envoy` already; do we need to re-qualify `Seconds`?

Reply:
Done.